libr/core/cmd_anal.c

```diff
@@ -8623,25 +8871,21 @@ static void cmd_anal_graph(RCore *core, const char *input) {
                         case 'k':
                         case 'v':
                         case 'g': {
-                                ut64 addr = input[2] ? r_num_math (core->num, input + 2) : core->offset;
+                                ut64 addr = input[2]? r_num_math (core->num, input + 2): core->offset;
                                 r_core_cmdf (core, "ag-; .agd* @ %"PFMT64u"; agg%s;", addr, input + 1);
                                 break;
                         }
                         case 'd': {
-                                ut64 addr = input[2] ? r_num_math (core->num, input + 2) : core->offset;
+                                ut64 addr = input[2]? r_num_math (core->num, input + 2): core->offset;
                                 r_core_gdiff_fcn (core, addr, core->offset);
                                 r_core_anal_graph (core, addr, diff_opt);
                                 break;
                         }
                         case 'w': {
                                 char *cmdargs = r_str_newf ("agdd 0x%"PFMT64x, core->offset);
-                                char *cmd = graph_cmd (core, cmdargs, input + 2);
-                                if (cmd && *cmd) {
-                                        r_core_cmd0(core, cmd);
-                                }
-                                free(cmd);
-                                free(cmdargs);
-                                break;
+                                convert_dotcmd_to_image (core, cmdargs, input + 2);
+				free (cmdargs);
```

Comment:
tabs

Reply:
What's exactly you suggestion? Keep spaces for this block or reindent it whole?

Reply:
Oh damn... it's all messed up... I thought this was "the wrong one", instead this is the right one and all others are wrong :D I would say to just make it wrong as well :PP